tests/mock/clitools.rs

```diff
@@ -28,31 +28,31 @@ use crate::mock::topical_doc_data;
 use crate::mock::{MockComponentBuilder, MockFile, MockInstallerBuilder};
 
 /// The configuration used by the tests in this module
-pub struct Config {
+pub(crate) struct Config {
     /// Where we put the rustup / rustc / cargo bins
-    pub exedir: PathBuf,
+    pub(crate) exedir: PathBuf,
```

Comment:
we shouldn't do this - just the pub(crate) on the struct, per our discussions and that bug.

Reply:
No, if we remove the modifier, it will be private. The bug fixed is the use of pub in pub(crate).

Reply:
Could this be another bug?

Reply:
I'm not suggesting private, I'm suggesting simple `pub`

Reply:
Oh, you mean in this case we don't need to change it to pub(crate) but keep it pub?